sources/packages/web/src/components/common/InstitutionUserSummary.vue

```diff
@@ -257,12 +290,18 @@ export default {
       }
     };
 
+    const canDisableUser = (userName: string): boolean => {
```

Comment:
It is a method called from inside the data table row, which means once per row. If the data associated with the row changes it will be called again.  I did not find anywhere a benefit of using a computed for this situation. If you have some, please let me know 😉

Reply:
Yes, it makes sense for it to be called once per row. Thanks for the clarification.